### PR TITLE
Fix docstring errors in stream.py, pipe.py, blockpartition.py, microbatch.py, namespace.py, profile.py, tracker.py, portal.py, layout.py, __init__.py

### LINKED ISSUE #112646: Fix docstring errors in stream.py, pipe.py, blockpartition.py, microbatch.py, namespace.py, profile.py, tracker.py, portal.py, layout.py, __init__.py
Please fix the following issues. To test locally, run: 
 ```
pydocstyle path-to-file --count
```
Run this command before and after fixing the errors. This command prints out the number of errors at the end of the output. Paste the output to your PR description with the number before and after. 
- **File**: `torch/distributed/pipeline/sync/microbatch.py`, **Entity**: `NoChunk`, **Line**: 24, **Description**: 1 blank line required after class docstring (found 0)
- **File**: `torch/distributed/pipeline/sync/microbatch.py`, **Entity**: `NoChunk`, **Line**: 24, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/distributed/pipeline/sync/microbatch.py`, **Entity**: `NoChunk`, **Line**: 24, **Description**: First line should end with a period (not 'r')
- **File**: `torch/distributed/pipeline/sync/microbatch.py`, **Entity**: `Batch`, **Line**: 41, **Description**: One-line docstring should fit on one line with quotes (found 3)
- **File**: `torch/distributed/pipeline/sync/microbatch.py`, **Entity**: `values`, **Line**: 63, **Description**: First line should end with a period (not 'h')
- **File**: `torch/distributed/pipeline/sync/microbatch.py`, **Entity**: `find_tensor_idx`, **Line**: 67, **Description**: One-line docstring should fit on one line with quotes (found 3)
- **File**: `torch/distributed/pipeline/sync/microbatch.py`, **Entity**: `find_tensor_idx`, **Line**: 67, **Description**: First line should be in imperative mood (perhaps 'Retrieve', not 'Retrieves')
- **File**: `torch/distributed/pipeline/sync/microbatch.py`, **Entity**: `get_device`, **Line**: 79, **Description**: One-line docstring should fit on one line with quotes (found 3)
- **File**: `torch/distributed/pipeline/sync/microbatch.py`, **Entity**: `get_device`, **Line**: 79, **Description**: First line should be in imperative mood (perhaps 'Retrieve', not 'Retrieves')
- **File**: `torch/distributed/pipeline/sync/microbatch.py`, **Entity**: `call`, **Line**: 90, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/distributed/pipeline/sync/microbatch.py`, **Entity**: `call`, **Line**: 90, **Description**: First line should end with a period (not 's')
- **File**: `torch/distributed/pipeline/sync/microbatch.py`, **Entity**: `call`, **Line**: 90, **Description**: First line should be in imperative mood (perhaps 'Call', not 'Calls')
- **File**: `torch/distributed/pipeline/sync/microbatch.py`, **Entity**: `check`, **Line**: 160, **Description**: No blank lines allowed after function docstring (found 1)
- **File**: `torch/distributed/pipeline/sync/microbatch.py`, **Entity**: `check`, **Line**: 160, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/distributed/pipeline/sync/microbatch.py`, **Entity**: `check`, **Line**: 160, **Description**: First line should end with a period (not 's')
- **File**: `torch/distributed/pipeline/sync/microbatch.py`, **Entity**: `check`, **Line**: 160, **Description**: First line should be in imperative mood (perhaps 'Check', not 'Checks')
- **File**: `torch/distributed/pipeline/sync/microbatch.py`, **Entity**: `scatter`, **Line**: 176, **Description**: First line should be in imperative mood (perhaps 'Split', not 'Splits')
- **File**: `torch/distributed/pipeline/sync/pipe.py`, **Entity**: `_recommend_auto_balance`, **Line**: 43, **Description**: First line should be in imperative mood (perhaps 'Expand', not 'Expands')
- **File**: `torch/distributed/pipeline/sync/pipe.py`, **Entity**: `_retrieve_device`, **Line**: 95, **Description**: No blank lines allowed after function docstring (found 1)
- **File**: `torch/distributed/pipeline/sync/pipe.py`, **Entity**: `_retrieve_device`, **Line**: 95, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/distributed/pipeline/sync/pipe.py`, **Entity**: `_retrieve_device`, **Line**: 95, **Description**: First line should end with a period (not 's')
- **File**: `torch/distributed/pipeline/sync/pipe.py`, **Entity**: `_retrieve_device`, **Line**: 95, **Description**: First line should be in imperative mood (perhaps 'Validate', not 'Validates')
- **File**: `torch/distributed/pipeline/sync/pipe.py`, **Entity**: `PipeSequential`, **Line**: 122, **Description**: One-line docstring should fit on one line with quotes (found 3)
- **File**: `torch/distributed/pipeline/sync/pipe.py`, **Entity**: `WithDevice`, **Line**: 137, **Description**: 1 blank line required after class docstring (found 0)
- **File**: `torch/distributed/pipeline/sync/pipe.py`, **Entity**: `WithDevice`, **Line**: 137, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/distributed/pipeline/sync/pipe.py`, **Entity**: `WithDevice`, **Line**: 137, **Description**: First line should end with a period (not '`')
- **File**: `torch/distributed/pipeline/sync/pipe.py`, **Entity**: `Pipe`, **Line**: 225, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/distributed/pipeline/sync/pipe.py`, **Entity**: `Pipe`, **Line**: 225, **Description**: First line should end with a period (not 'e')
- **File**: `torch/distributed/pipeline/sync/pipe.py`, **Entity**: `__len__`, **Line**: 358, **Description**: First line should be in imperative mood (perhaps 'Count', not 'Counts')
- **File**: `torch/distributed/pipeline/sync/pipe.py`, **Entity**: `__getitem__`, **Line**: 362, **Description**: First line should be in imperative mood (perhaps 'Get', not 'Gets')
- **File**: `torch/distributed/pipeline/sync/pipe.py`, **Entity**: `__iter__`, **Line**: 383, **Description**: First line should be in imperative mood (perhaps 'Iterate', not 'Iterates')
- **File**: `torch/distributed/pipeline/sync/pipe.py`, **Entity**: `_ensure_copy_streams`, **Line**: 417, **Description**: First line should be in imperative mood (perhaps 'Ensure', not 'Ensures')
- **File**: `torch/distributed/pipeline/sync/pipe.py`, **Entity**: `forward`, **Line**: 431, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/distributed/pipeline/sync/pipe.py`, **Entity**: `forward`, **Line**: 431, **Description**: First line should end with a period (not 'n')
- **File**: `torch/distributed/pipeline/sync/pipe.py`, **Entity**: `forward`, **Line**: 431, **Description**: First line should be in imperative mood (perhaps 'Process', not 'Processes')
- **File**: `torch/distributed/pipeline/sync/stream.py`, **Entity**: ``, **Line**: 7, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/distributed/pipeline/sync/stream.py`, **Entity**: ``, **Line**: 7, **Description**: First line should end with a period (not 'h')
- **File**: `torch/distributed/pipeline/sync/stream.py`, **Entity**: `new_stream`, **Line**: 32, **Description**: First line should be in imperative mood (perhaps 'Create', not 'Creates')
- **File**: `torch/distributed/pipeline/sync/stream.py`, **Entity**: `get_device`, **Line**: 75, **Description**: First line should be in imperative mood (perhaps 'Get', not 'Gets')
- **File**: `torch/distributed/pipeline/sync/stream.py`, **Entity**: `wait_stream`, **Line**: 82, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/distributed/pipeline/sync/stream.py`, **Entity**: `wait_stream`, **Line**: 82, **Description**: First line should end with a period (not 't')
- **File**: `torch/distributed/pipeline/sync/stream.py`, **Entity**: `is_cuda`, **Line**: 114, **Description**: First line should be in imperative mood (perhaps 'Return', not 'Returns')
- **File**: `torch/distributed/pipeline/sync/_balance/blockpartition.py`, **Entity**: `solve`, **Line**: 19, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/distributed/pipeline/sync/_balance/blockpartition.py`, **Entity**: `solve`, **Line**: 19, **Description**: First line should end with a period (not 'h')
- **File**: `torch/distributed/pipeline/sync/_balance/blockpartition.py`, **Entity**: `solve`, **Line**: 19, **Description**: First line should be in imperative mood (perhaps 'Split', not 'Splits')
- **File**: `torch/distributed/pipeline/sync/_balance/__init__.py`, **Entity**: `balance_by_time`, **Line**: 52, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/distributed/pipeline/sync/_balance/profile.py`, **Entity**: `layerwise_sandbox`, **Line**: 28, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/distributed/pipeline/sync/_balance/profile.py`, **Entity**: `layerwise_sandbox`, **Line**: 28, **Description**: First line should end with a period (not 'n')
- **File**: `torch/distributed/pipeline/sync/_balance/profile.py`, **Entity**: `layerwise_sandbox`, **Line**: 28, **Description**: First line should be in imperative mood (perhaps 'Copy', not 'Copies')
- **File**: `torch/distributed/pipeline/sync/skip/namespace.py`, **Entity**: `Namespace`, **Line**: 18, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/distributed/pipeline/sync/skip/namespace.py`, **Entity**: `Namespace`, **Line**: 18, **Description**: First line should end with a period (not ')')
- **File**: `torch/distributed/pipeline/sync/skip/portal.py`, **Entity**: ``, **Line**: 8, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/distributed/pipeline/sync/skip/portal.py`, **Entity**: ``, **Line**: 8, **Description**: First line should end with a period (not 'e')
- **File**: `torch/distributed/pipeline/sync/skip/portal.py`, **Entity**: `blue`, **Line**: 37, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/distributed/pipeline/sync/skip/portal.py`, **Entity**: `blue`, **Line**: 37, **Description**: First line should end with a period (not 'm')
- **File**: `torch/distributed/pipeline/sync/skip/portal.py`, **Entity**: `blue`, **Line**: 37, **Description**: First line should be in imperative mood (perhaps 'Create', not 'Creates')
- **File**: `torch/distributed/pipeline/sync/skip/portal.py`, **Entity**: `orange`, **Line**: 56, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/distributed/pipeline/sync/skip/portal.py`, **Entity**: `orange`, **Line**: 56, **Description**: First line should end with a period (not 'r')
- **File**: `torch/distributed/pipeline/sync/skip/portal.py`, **Entity**: `orange`, **Line**: 56, **Description**: First line should be in imperative mood (perhaps 'Create', not 'Creates')
- **File**: `torch/distributed/pipeline/sync/skip/portal.py`, **Entity**: `copy`, **Line**: 74, **Description**: First line should be in imperative mood (perhaps 'Copy', not 'Copies')
- **File**: `torch/distributed/pipeline/sync/skip/portal.py`, **Entity**: `put_tensor`, **Line**: 93, **Description**: First line should be in imperative mood (perhaps 'Store', not 'Stores')
- **File**: `torch/distributed/pipeline/sync/skip/portal.py`, **Entity**: `use_tensor`, **Line**: 126, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/distributed/pipeline/sync/skip/portal.py`, **Entity**: `use_tensor`, **Line**: 126, **Description**: First line should end with a period (not 'n')
- **File**: `torch/distributed/pipeline/sync/skip/portal.py`, **Entity**: `use_tensor`, **Line**: 126, **Description**: First line should be in imperative mood (perhaps 'Retrieve', not 'Retrieves')
- **File**: `torch/distributed/pipeline/sync/skip/portal.py`, **Entity**: `put_grad`, **Line**: 141, **Description**: First line should be in imperative mood (perhaps 'Store', not 'Stores')
- **File**: `torch/distributed/pipeline/sync/skip/portal.py`, **Entity**: `use_grad`, **Line**: 145, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/distributed/pipeline/sync/skip/portal.py`, **Entity**: `use_grad`, **Line**: 145, **Description**: First line should end with a period (not 's')
- **File**: `torch/distributed/pipeline/sync/skip/portal.py`, **Entity**: `use_grad`, **Line**: 145, **Description**: First line should be in imperative mood (perhaps 'Retrieve', not 'Retrieves')
- **File**: `torch/distributed/pipeline/sync/skip/portal.py`, **Entity**: `PortalCopy`, **Line**: 207, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/distributed/pipeline/sync/skip/portal.py`, **Entity**: `PortalCopy`, **Line**: 207, **Description**: First line should end with a period (not 'n')
- **File**: `torch/distributed/pipeline/sync/skip/layout.py`, **Entity**: `copy_policy`, **Line**: 40, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/distributed/pipeline/sync/skip/layout.py`, **Entity**: `copy_policy`, **Line**: 40, **Description**: First line should be in imperative mood (perhaps 'Generate', not 'Generates')
- **File**: `torch/distributed/pipeline/sync/skip/layout.py`, **Entity**: `requires_copy`, **Line**: 57, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/distributed/pipeline/sync/skip/layout.py`, **Entity**: `requires_copy`, **Line**: 57, **Description**: First line should end with a period (not 'n')
- **File**: `torch/distributed/pipeline/sync/skip/tracker.py`, **Entity**: `SkipTrackerThroughPotals`, **Line**: 53, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/distributed/pipeline/sync/skip/tracker.py`, **Entity**: `SkipTrackerThroughPotals`, **Line**: 53, **Description**: First line should end with a period (not 'e')
- **File**: `torch/distributed/pipeline/sync/skip/tracker.py`, **Entity**: `save`, **Line**: 67, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/distributed/pipeline/sync/skip/tracker.py`, **Entity**: `save`, **Line**: 67, **Description**: First line should end with a period (not 'n')
- **File**: `torch/distributed/pipeline/sync/skip/tracker.py`, **Entity**: `save`, **Line**: 67, **Description**: First line should be in imperative mood (perhaps 'Save', not 'Saves')
- **File**: `torch/distributed/pipeline/sync/skip/tracker.py`, **Entity**: `load`, **Line**: 114, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/distributed/pipeline/sync/skip/tracker.py`, **Entity**: `load`, **Line**: 114, **Description**: First line should end with a period (not 'n')
- **File**: `torch/distributed/pipeline/sync/skip/tracker.py`, **Entity**: `load`, **Line**: 114, **Description**: First line should be in imperative mood (perhaps 'Load', not 'Loads')
- **File**: `torch/distributed/pipeline/sync/skip/tracker.py`, **Entity**: `copy`, **Line**: 130, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/distributed/pipeline/sync/skip/tracker.py`, **Entity**: `copy`, **Line**: 130, **Description**: First line should end with a period (not 'n')
- **File**: `torch/distributed/pipeline/sync/skip/tracker.py`, **Entity**: `copy`, **Line**: 130, **Description**: First line should be in imperative mood (perhaps 'Copy', not 'Copies')
- **File**: `torch/distributed/pipeline/sync/skip/tracker.py`, **Entity**: `use_skip_tracker`, **Line**: 155, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/distributed/pipeline/sync/skip/tracker.py`, **Entity**: `use_skip_tracker`, **Line**: 155, **Description**: First line should end with a period (not 'a')
- **File**: `torch/distributed/pipeline/sync/skip/tracker.py`, **Entity**: `use_skip_tracker`, **Line**: 155, **Description**: First line should be in imperative mood (perhaps 'Register', not 'Registers')
- **File**: `torch/distributed/pipeline/sync/skip/tracker.py`, **Entity**: `current_skip_tracker`, **Line**: 173, **Description**: First line should be in imperative mood (perhaps 'Get', not 'Gets')

### PR DESCRIPTION
**Pull request by sitammeur**
Fixes #112646

```
pydocstyle torch/distributed/pipeline/sync/microbatch.py --count
```
On master: 17
After my changes to this PR: 0

```
pydocstyle torch/distributed/pipeline/sync/pipe.py --count
```
On master: 18
After my changes on this PR: 0

```
pydocstyle torch/distributed/pipeline/sync/stream.py --count
```
On master: 7
After my changes on this PR: 0

```
pydocstyle torch/distributed/pipeline/sync/_balance/blockpartition.py --count
```
On master: 3
After my changes on this PR: 0

```
pydocstyle torch/distributed/pipeline/sync/_balance/__init__.py --count
```
On master: 1
After my changes on this PR: 0

```
pydocstyle torch/distributed/pipeline/sync/_balance/profile.py --count
```
On master: 3
After my changes on this PR: 0

```
pydocstyle torch/distributed/pipeline/sync/skip/namespace.py--count
```
On master: 2
After my changes on this PR: 0

```
pydocstyle torch/distributed/pipeline/sync/skip/portal.py  --count
```
On master: 19
After my changes on this PR: 0

```
pydocstyle torch/distributed/pipeline/sync/skip/layout.py --count
```
On master: 4
After my changes to this PR: 0

```
pydocstyle torch/distributed/pipeline/sync/skip/tracker.py --count
```
On master: 15
After my changes on this PR: 0

@svekars 

Note: In the error count, I have not included the make docs strings error; only the errors mentioned in the issue are mentioned.

cc @svekars @brycebortree @carljparker

### CODE REVIEW

File: torch/distributed/pipeline/sync/microbatch.py [hunk 1]
```diff
@@ -6,11 +6,11 @@
 # LICENSE file in the root directory of this source tree.
 """Manipulation of micro-batches."""
 import typing
-from typing import Any, Callable, List, Union, cast, Sequence
+from typing import Any, Callable, cast, List, Sequence, Union
```
Comment (svekars) [inline torch/distributed/pipeline/sync/microbatch.py:9]: why are you changing these?

File: torch/distributed/pipeline/sync/microbatch.py [hunk 2]
```diff
@@ -187,7 +184,9 @@ def scatter(*inputs, chunks: int) -> List[Batch]:
 
             # Validate number of chunks equal across all inputs.
             if num_chunks != -1 and num_chunks != len(tensors):
-                raise RuntimeError(f'Found different number of chunks produced for inputs: {num_chunks} and {len(tensors)}')
+                raise RuntimeError(
+                    f"Found different number of chunks produced for inputs: {num_chunks} and {len(tensors)}"
```
Comment (svekars) [inline torch/distributed/pipeline/sync/microbatch.py:188]: I don't believe this needs to be modified

File: torch/distributed/pipeline/sync/pipe.py [hunk 3]
```diff
@@ -71,7 +82,9 @@ def _verify_splitting(
     module: nn.Sequential, partitions: List[nn.Sequential], devices: List[torch.device]
 ) -> None:
     num_parameters = len(list(module.parameters()))
-    num_child_parameters = sum(len(list(child.parameters())) for child in module.children())
+    num_child_parameters = sum(
```
Comment (svekars) [inline torch/distributed/pipeline/sync/pipe.py:85]: this does not have to modified

**Review by svekars — COMMENTED** [review]
It looks like your editor makes changes to the file that were not requested. I marked some of them - can you please make sure you remove these changes?

**Comment by sitamgithub-MSIT** [comment]
@svekars  Sorry for the inconvenience. You said rightly that there were formatting problems due to my code editor not being intentionally changed. Anyway, I fixed all of them that you mentioned. Can you review that or any other things that need to be changed? I will surely do that.

File: torch/distributed/pipeline/sync/microbatch.py [hunk 4]
```diff
@@ -158,22 +155,22 @@ def _setitem_by_slice(self, index: slice, value) -> None:
...
 
     Raises:
-        ValueError: input does not contain at least one tensor
+        ValueError: input does not contain at least one tensor.
 
     """
-
     if not any(torch.is_tensor(input) for input in inputs):
-        raise TypeError(f'inputs do not have any tensors: {inputs}')
+        raise TypeError(f"inputs do not have any tensors: {inputs}")
```
Comment (svekars) [inline torch/distributed/pipeline/sync/microbatch.py:165]: please undo these changes

File: torch/distributed/pipeline/sync/microbatch.py [hunk 5]
```diff
@@ -158,22 +155,22 @@ def _setitem_by_slice(self, index: slice, value) -> None:
...
     if any(torch.is_tensor(input) and input.device != first_device for input in inputs):
-        raise ValueError('All inputs should be on the same device as the first partition')
+        raise ValueError(
+            "All inputs should be on the same device as the first partition"
+        )
```
Comment (svekars) [inline torch/distributed/pipeline/sync/microbatch.py:169]: please undo these changes

File: torch/distributed/pipeline/sync/pipe.py [hunk 6]
```diff
@@ -9,10 +9,10 @@
 from typing import TYPE_CHECKING, Any, Iterable, Iterator, List, Optional, Union, Sequence, Tuple, cast
 
 import torch
-from torch import Tensor, nn
-from torch.distributed.rpc import RRef
 import torch.autograd
 import torch.cuda
+from torch import nn, Tensor
+from torch.distributed.rpc import RRef
```
Comment (svekars) [inline torch/distributed/pipeline/sync/pipe.py:15]: please undo these changes

File: torch/distributed/pipeline/sync/pipe.py [hunk 7]
```diff
@@ -135,11 +131,10 @@ def forward(self, *inputs):
 
 class WithDevice(nn.Module):
     """
-    Wraps an ``nn.Module`` which is part of ``nn.Sequential`` passed into :class:`Pipe`
-    that overrides the device for that module. In cases where :class:`Pipe`
-    can't implicitly determine the device for the module and places it on CPU,
-    this wrapper can be used to override the implicit behavior and explicitly
-    specify which device a module should run on.
+    Wraps an ``nn.Module`` which is part of ``nn.Sequential`` passed into :class:`Pipe`that overrides the device for that module.
```
Comment (svekars) [inline torch/distributed/pipeline/sync/pipe.py:134]:
```suggestion
    Wraps an ``nn.Module`` which is part of ``nn.Sequential`` passed into :class:`Pipe` that overrides the device for that module.
```

File: torch/distributed/pipeline/sync/pipe.py [hunk 8]
```diff
@@ -135,11 +131,10 @@ def forward(self, *inputs):
...
+
+    In cases where :class:`Pipe`can't implicitly determine the device for the module and places it on CPU,this wrapper can be used
```
Comment (svekars) [inline torch/distributed/pipeline/sync/pipe.py:136]:
```suggestion
    In cases where :class:`Pipe` can't implicitly determine the device for the module and places it on CPU, this wrapper can be used
```

File: torch/distributed/pipeline/sync/pipe.py [hunk 9]
```diff
@@ -202,7 +198,7 @@ def _split_module(modules: nn.Sequential) -> Tuple[List[nn.Sequential], List[tor
             module.to(device)
         else:
             device = _retrieve_device(module)
-        if current_device is not None and (current_device != device or device.type == 'cpu'):
+        if current_device is not None and (current_device != device or device.type == "cpu"):
```
Comment (svekars) [inline torch/distributed/pipeline/sync/pipe.py:201]: please undo these changes

File: torch/distributed/pipeline/sync/pipe.py [hunk 10]
```diff
@@ -326,7 +322,9 @@ def __init__(
         if chunks <= 0:
             raise ValueError("number of chunks must be positive integer")
         if checkpoint not in ["always", "except_last", "never"]:
-            raise ValueError("checkpoint is not one of 'always', 'except_last', or 'never'")
+            raise ValueError(
+                "checkpoint is not one of 'always', 'except_last', or 'never'"
+            )
```
Comment (svekars) [inline torch/distributed/pipeline/sync/pipe.py:327]: please undo these changes

File: torch/distributed/pipeline/sync/pipe.py [hunk 11]
```diff
@@ -350,16 +348,26 @@ def __init__(
...
+            "never": 0,
+        }[self.checkpoint]
+
+        self.pipeline = Pipeline(
+            self.partitions,
+            self.devices,
+            copy_streams,
+            self._skip_layout,
+            checkpoint_stop,
+        )
```
Comment (svekars) [inline torch/distributed/pipeline/sync/pipe.py:363]: please undo these changes

File: torch/distributed/pipeline/sync/pipe.py [hunk 12]
```diff
@@ -423,13 +431,16 @@ def _ensure_copy_streams(self) -> List[List[AbstractStream]]:
         """
         if not self._copy_streams:
             for device in self.devices:
-                self._copy_streams.append([new_stream(device) for _ in range(self.chunks)])
+                self._copy_streams.append(
+                    [new_stream(device) for _ in range(self.chunks)]
+                )
```
Comment (svekars) [inline torch/distributed/pipeline/sync/pipe.py:436]: please undo these changes

File: torch/distributed/pipeline/sync/pipe.py [hunk 13]
```diff
@@ -472,7 +483,9 @@ def forward(self, *inputs) -> RRef:
             TypeError: input doesn't contain at least one tensor
 
         """
-        first_partition_device = self.devices[0] if len(self.devices) != 0 else torch.device("cpu")
+        first_partition_device = (
+            self.devices[0] if len(self.devices) != 0 else torch.device("cpu")
+        )
```
Comment (svekars) [inline torch/distributed/pipeline/sync/pipe.py:488]: please undo these changes

File: torch/distributed/pipeline/sync/skip/namespace.py [hunk 14]
```diff
@@ -15,9 +15,7 @@
 
 @total_ordering
 class Namespace(metaclass=abc.ABCMeta):
-    """Namespace for isolating skip tensors used by :meth:`isolate()
-    <torchpipe.skip.skippable.Skippable.isolate>`.
-    """
+    """Namespace for isolating skip tensors used by :meth:`isolate() <torchpipe.skip.skippable.Skippable.isolate>`."""
```
Comment (svekars) [inline torch/distributed/pipeline/sync/skip/namespace.py:18]:
```suggestion
    """A namespace for isolating skip tensors used by :meth:`isolate() <torchpipe.skip.skippable.Skippable.isolate>`."""
```

File: torch/distributed/pipeline/sync/skip/portal.py [hunk 15]
```diff
@@ -122,8 +121,9 @@ def put_tensor(self, tensor: Optional[Tensor], tensor_life: int) -> None:
             self.tensor = None
 
     def use_tensor(self) -> Optional[Tensor]:
-        """Retrieves the underlying tensor and decreases the tensor  life. When
-        the life becomes 0, it the tensor will be removed.
+        """Retrieve the underlying tensor and decreases the tensor  life.
```
Comment (svekars) [inline torch/distributed/pipeline/sync/skip/portal.py:124]:
```suggestion
        """Retrieve the underlying tensor and decreases the tensor life.
```

File: torch/distributed/pipeline/sync/skip/portal.py [hunk 16]
```diff
@@ -137,12 +137,13 @@ def use_tensor(self) -> Optional[Tensor]:
...
 
     def put_grad(self, grad: Tensor) -> None:
-        """Stores a gradient into this portal."""
+        """Store a gradient into this portal."""
         self.grad = grad
 
     def use_grad(self) -> Tensor:
-        """Retrieves and removes the underlying gradient. The gradient is
-        always ephemeral.
+        """Retrieve and removes the underlying gradient.
```
Comment (svekars) [inline torch/distributed/pipeline/sync/skip/portal.py:144]:
```suggestion
        """Retrieve and remove the underlying gradient.
```

**Comment by sitamgithub-MSIT** [comment]
@svekars All the changes were done as you had mentioned. Please review it.

File: torch/distributed/pipeline/sync/skip/tracker.py [hunk 17]
```diff
@@ -152,11 +156,10 @@ def __init__(self) -> None:
...
 @contextmanager
 def use_skip_tracker(skip_tracker: SkipTracker) -> Generator[None, None, None]:
-    """Registers the given skip tracker on the current thread within a
-    context::
+    """Register the given skip tracker on the current thread within a context.
 
-        with use_skip_tracker(my_skip_tracker):
-            ...
+    with use_skip_tracker(my_skip_tracker):
+        ...
```
Comment (svekars) [inline torch/distributed/pipeline/sync/skip/tracker.py:162]: please undo this change

**Comment by sitamgithub-MSIT** [comment]
@svekars Updates made!

**Comment by svekars** [comment]
Can you please fix linter?

**Comment by sitamgithub-MSIT** [comment]
Can you please fix Linter?

Sure, I will fix that. Apart from Linter, one check also failed. Is that also a problem?

**Comment by sitamgithub-MSIT** [comment]
@svekars Can you review it and tell me if any changes are needed or not?
